# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/glibc/hotfix-gnu23.diff
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- glibc-2.43/include/features.h.vanilla	2026-01-25 00:53:27.142617648 +0100
+++ glibc-2.43/include/features.h	2026-01-25 00:54:13.941812010 +0100
@@ -223,9 +223,9 @@
 # undef  _ISOC11_SOURCE
 # define _ISOC11_SOURCE	1
 # undef  _ISOC23_SOURCE
-# define _ISOC23_SOURCE	1
+/*# define _ISOC23_SOURCE	1*/
 # undef  _ISOC2Y_SOURCE
-# define _ISOC2Y_SOURCE	1
+/*# define _ISOC2Y_SOURCE	1*/
 # undef  _POSIX_SOURCE
 # define _POSIX_SOURCE	1
 # undef  _POSIX_C_SOURCE
